backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientMutationResolver.java

```diff
@@ -19,13 +29,21 @@
 public class PatientMutationResolver implements GraphQLMutationResolver  {
 
     private final PersonService _ps;
-  
-    public PatientMutationResolver(PersonService ps) {
+    private final UploadService _us;
+
+    public PatientMutationResolver(PersonService ps, UploadService us) {
         _ps = ps;
+        _us = us;
+    }
+
+    @AuthorizationConfiguration.RequireGlobalAdminUser
+    public String uploadPatients(Part part) throws Exception {
```

Comment:
I think we're trying to attach the authorization tags at the service layer whenever possible, for consistency.